spec/features/job_seekers_can_see_a_map_spec.rb

```diff
@@ -13,7 +13,7 @@
 
   scenario 'should not display a map when a school has no geocoding' do
     school = FactoryGirl.create(:school)
-    vacancy = FactoryGirl.create(:vacancy, school: school)
+    vacancy = FactoryGirl.create(:vacancy, :complete, school: school)
```

Comment:
You've moved these fields in the factory out of the `complete` scope so this addition looks like it can be undone?